sonic_sfp/sfputilbase.py

```diff
@@ -268,15 +310,14 @@ def read_porttab_mappings(self, porttabfile):
 
         # Read the porttab file and generate dicts
         # with mapping for future reference.
-        #
-        # TODO: Refactor this to use the portconfig.py module that now
-        # exists as part of the sonic-config-engine package.
-        title = []
+        # XXX: move the porttab
+        # parsing stuff to a separate module, or reuse
+        # if something already exists
```

Comment:
Please change this comment back, as the other comment is newer and more descriptive as to what needs to be done.